src/sentry/snuba/functions.py

```diff
@@ -270,7 +272,7 @@ def format_top_events_timeseries_results(
                         else item["data"]
                     ),
                     "order": item["order"],
-                    "meta": result["meta"],
+                    "meta": processed_result["meta"],
```

Comment:
@Zylphrex heads-up! Before this change, I think `"meta"` had the unprocessed Snuba meta response. Now it'll have Discover's meta. I assume this is safe, but I'm not sure if the FE uses this!

Reply:
This should be okay 👍